test/smoke/src/areas/positron/editor/fast-execution.test.ts

```diff
@@ -30,12 +30,23 @@ export function setup(logger: Logger) {
 
 			});
 
-			it('Verify fast execution is not out of order [C712539]', async function () {
+			it('Verify fast execution is not out of order [C712539] #pr', async function () {
 				const app = this.app as Application;
 
 				await app.workbench.quickaccess.openFile(join(app.workspacePathOrFolder, 'workspaces', 'fast-statement-execution', FILENAME));
 
+				let previousTop = -1;
 				for (let i = 1; i < 12; i++) {
```

Comment:
Perhaps a comment to explain advancing 11 lines (< 12) is because you're trying to exhaustively run all of the 10 lines in the test case .R file?

Reply:
Maybe also mention that top here is a position measurement being compared to determine cursor line advancement in the editor?

Reply:
Yep.  Added comments for both!